src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -185,15 +185,15 @@
                 </v-btn>
 
                 <!-- the drop-down menu -->
-                <v-menu offset-y left transition="slide-y-transition" v-if="isRoleStaff && !tempRegNumber">
+                <v-menu offset-y left transition="slide-y-transition" v-if="isRoleStaff && !!businessId">
```

Comment:
`tempRegNumber` and `businessId` are mutually exclusive: we either get one or the other.

I changed this line of code because I think it makes more sense to "do this if we have a businessId" (instead of "do this if we don't have a tempRegNumber").

Reply:
Right on.
I think we did the same in the create ui. A common computed value.

Reply:
By the way, I'm using `!!` something more and more to convert from a non-boolean to a boolean. Like here, where "businessId" is a string.